README.md

```diff
@@ -2,15 +2,38 @@
 
 ## Introduction
 
-This repo provides the code for several example dApps in the [Axelar Local Development Environment](https://github.com/axelarnetwork/axelar-local-dev). Examples contain both JavaScript and Solidity smart contract code.
+Welcome to Axelar cross-chain dApp examples! This repository provides the code for several example dApps in the [Axelar Local Development Environment](https://github.com/axelarnetwork/axelar-local-dev). These examples contain both JavaScript and Solidity smart contract code that can be used as a starting point for building your own cross-chain dApps. This readme provides a detailed guide on how to set up the development environment and run the examples.
```

Comment:
I think we should call it `axelar-examples` or "Axelar Examples" to keep the naming consistent.  Axelar cross-chain dApp examples is a description, not a name.

Reply:
Updated